### PR TITLE
Issue-1: Add aliases from .gitaliases file

### LINKED ISSUE #1: Add aliases from .gitaliases file

### PR DESCRIPTION
**Pull request by jlmorgan**
Closes #1
SemVer:Minor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+1. Add the following to your execution `PATH` in wherever you manage your execution `PATH`:
+```bash
+# Don't seriously copy-paste and be done. You will need to update the path.
+export PATH="${PATH}:~/path/to/git-extras.git/bin"
+```
+1. Reload your session, profile file, etc.
+
+### Optional Bits
+
+The following commands have a dependency on an external project to do things and stuff and junk. If the dependency is not install, it will present an error.
```
Comment (jbatte47) [inline README.md]: typo: "install" &rarr; "installed"

File: bin/git-bugfix-from-current [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+set -o errexit
+set -o errtrace
+set -o pipefail
+
+# shellcheck source=/dev/null
+source "$(git --exec-path)/git-sh-setup"
+
+function _bugfix_from_current() {
+  # shellcheck disable=SC2034
+  local USAGE="<tickedId>"
```
Comment (jbatte47) [inline bin/git-bugfix-from-current]: typo? `ticked` &rarr; `ticket` ?

Comment (jlmorgan) [inline bin/git-bugfix-from-current]: Fixed.

File: README.md (hunk 1 above)
Comment (jlmorgan) [inline README.md]: Fixed.

**Pull request merged**